pkg/network/ovn_kubernetes.go

```diff
@@ -82,6 +82,11 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["OVN_NB_DB_LIST"] = dbList(bootstrapResult.OVN.MasterIPs, OVN_NB_PORT)
 	data.Data["OVN_SB_DB_LIST"] = dbList(bootstrapResult.OVN.MasterIPs, OVN_SB_PORT)
 	data.Data["OVN_DB_CLUSTER_INITIATOR"] = bootstrapResult.OVN.ClusterInitiator
+	if len(bootstrapResult.OVN.MasterIPs) == 1 {
```

Comment:
As pointed out in the bug dispatch meeting, we seem to be doing this "len(bootstrapResult.OVN.MasterIPs)" call multiple times in the function. Ref: https://github.com/openshift/cluster-network-operator/pull/1154/commits 
Maybe we should convert it ti a local variable?